contrib/vagrant/control-plane-base.sh

```diff
@@ -1,6 +1,161 @@
 #!/bin/bash
 
-zypper install -y -t pattern kubic_admin
-install -D -m 0644 /home/vagrant/enclave/contrib/crio/00-default.conf /etc/crio/crio.conf.d/00-default.conf
-systemctl restart crio
-systemctl enable kubelet.service
+# # zypper install -y -t pattern kubic_admin
+# zypper install -y cri-o
+# # # install -D -m 0644 /home/vagrant/enclave/contrib/crio/00-default.conf /etc/crio/crio.conf.d/00-default.conf
+# mv /etc/crio/crio.conf.d/00-default.conf /etc/crio/crio.conf
+# systemctl enable crio
+# systemctl start crio
+# # systemctl restart crio
+# # systemctl enable kubelet.service
+
+cat <<EOF > /etc/containerd/config.toml
+root = "/var/lib/containerd"
+state = "/run/containerd"
+oom_score = 0
+
+[grpc]
+  address = "/run/containerd/containerd.sock"
+  uid = 0
+  gid = 0
+  max_recv_message_size = 16777216
+  max_send_message_size = 16777216
+
+[debug]
+  address = ""
+  uid = 0
+  gid = 0
+  level = ""
+
+[metrics]
+  address = ""
+  grpc_histogram = false
+
+[cgroup]
+  path = ""
+
+[plugins]
+  [plugins.cgroups]
+    no_prometheus = false
+  [plugins.cri]
+    stream_server_address = ""
+    stream_server_port = "10010"
+    enable_selinux = false
+    sandbox_image = "k8s.gcr.io/pause:3.2"
```

Comment:
Do we want to change this to "latest" or "3" for flexibility? The vagrant image is tumbleweed, after all.

Reply:
I can try `latest`